### DISCUSSION TITLE
疑是ipv4、sni ip的复活些

### DISCUSSION THREAD

**Discussion by wpyok500** [SeaHOH/GotoX#91] [一般]
profile = ipv46

ip.use里的ip全是ipv4的地址，之前是ipv6的

**Comment by SeaHOH** [comment]
你使用的顺畅吗？我这里没发现有太大变化。

**Comment by wpyok500** [comment]
公司网络，没做测试，主要是浏览网页。

**Comment by SeaHOH** [comment]
没有关注 IPv4，据说恢复了部分 QUIC。
我自己不想花钱，用别人 SNI 又太伤人品，早就换 IPv6 了。

**Comment by SeaHOH** [comment]
虽然细节不准确，你这么理解也行。

**Comment by MeABc** [comment]
> 用别人 SNI 又太伤人品

老大，你这么说让我情何以堪。。。。

**Comment by SeaHOH** [comment]
Don't mind~ 我之前就用了一段时间，而且现在偶尔也用一下。 :smiley_cat:

**Comment by SeaHOH** [comment]
才注意到，我又被自言自语了。。。
